File: ch7/yojkim/ch7/franc_test.go
package ch7

import (
	"github.com/stretchr/testify/assert"
	"testing"
)

func TestFrancMultiplication(t *testing.T) {
	five := NewFranc(5)
	assert.Equal(t, NewFranc(10), *five.Times(2))
	assert.Equal(t, NewFranc(15), *five.Times(3))
}

File: ch5/gunddol/src/ch5/Franc_Test.java
package ch5;

import org.junit.Test;

import static org.junit.Assert.*;

public class Franc_Test {

    @Test
    public void testFrancMultiplication() {
        Franc five = new Franc(5);
        assertEquals(new Franc(10), five.times(2));
        assertEquals(new Franc(15), five.times(3));
    }

}